example-context/example-service/example-liquibase/src/main/resources/liquibase/view-store-db-changesets/001-initial-schema.changelog.xml

```diff
@@ -14,6 +14,9 @@
             <column name="name" type="TEXT">
                 <constraints nullable="false"/>
             </column>
+            <column name="glutenFree" type="boolean">
```

Comment:
Do we want to edit the initial file or add a new filter with an alter table as liquibase is intended to work?